### PR TITLE
Update all modules and Drupal Core

### PR DESCRIPTION
**Pull request by tonifisler**

### CODE REVIEW

File: drupal/.htaccess [hunk 1]
```diff
@@ -56,47 +56,6 @@ DirectoryIndex index.php index.html index.htm
...
-  Redirect 301 /en/job /en/we/are-hiring
-  Redirect 301 /fr/job/web-developer-drupal /fr/nous/embauchons
-  Redirect 301 /en/job/web-developer-drupal /en/we/are-hiring
-  Redirect 301 /en/contact /en/we/work/with/you
-  Redirect 301 /fr/contact /fr/nous/travaillons/avec/vous
-
-  # Redirect old Wordpress blogpost to new one
-  RewriteCond %{REQUEST_FILENAME} !-f
-  RewriteCond %{REQUEST_FILENAME} !-d
-  RewriteRule ^blog/(.+) /redirect_blog.php [L]
```
Comment (gido) [inline drupal/.htaccess]: @tonifisler all this redirections changes

**Comment by gido** [comment]
@tonifisler a mon avis il faut réappliquer mon ~~hack~~ patch sur le module `ace_editor`:

```
$ patch < patches/ace_editor_module_to_support_term.patch
```

**Comment by tonifisler** [comment]
Can't apply the patch. Everything changed in this `ace_editor_form_alter` method. I think you'll figure out more quickly what you wanted to do at the beginning.

**Comment by gido** [comment]
@tonifisler looks like they implemented (more nicely) a part of my patch in the new release of the module.

**Pull request merged**